app/javascript/components/breadcrumbs/index.jsx

```diff
@@ -45,14 +46,17 @@ const renderItems = ({ items, controllerName }) => {
 };
 
 const Breadcrumbs = ({ items, title, controllerName }) => (
```

Comment:
I woudln't call it just `Breadcrumbs` if we also put the notifications into it.

Reply:
left Breadcrumbs toggle-less and added BreadcrumbsBar to wrap both :+1: